steps/telegram-notify/v1.0.0/step.yml

```diff
@@ -0,0 +1,59 @@
+title: telegram_notify
```

Comment:
The `title` is the name of the step, which presents on our web UI, please do not use `_`. The name should look like: <action><service/tool> (in any order). I would recommend: `Send Telegram message`